samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml

```
@@ -39,6 +39,20 @@ spec:
         - |
           set -euo pipefail
 
+          echo "> Processing environment variables..."
+          ENV_DIR="/platform/env"
+
+          envs=($(env))
+
+          for env in "${envs[@]}"; do
+              IFS='=' read -r key value string <<< "$env"
+              if [[ "$key" != "" && "$value" != "" ]]; then
```

Let's not ignore empty values. There might be theoretic use cases where "not present" vs "empty string" makes a difference.

```suggestion
              if [[ "$key" != "" ]]; then
```